Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -625,29 +598,29 @@ namespace OpenInfraPlatform {
 					if(radius < 0.000001) {

```

Comment:
```suggestion
					if(radius < GeomSettings()->getPrecision()) {
```
We now have a function that tells what precision a file guaranties.

Reply:
Done